### PR TITLE
Feature/new product

### PR DESCRIPTION
**Pull request by Asergey91**
adds a button to create an empty product/service
service pr: https://github.com/lblod/lpdc-management-service/pull/4

### CODE REVIEW

**Comment by Windvis** [comment]
I assume the linting issues are caused by the target branch so we can ignore that for now.

File: app/components/public-services/details-page.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ export default class PublicServicesDetailsPageComponent extends Component {
     this.modals.open(ConfirmDeletionModal, {
       deleteHandler: async () => {
         await this.args.publicService.destroyRecord();
+        this.hasUnsavedChanges = false;
```
[the remark this answers is not in the document]
Comment (Asergey91) [reply app/components/public-services/details-page.js:109]: yep thats specifically for that, didnt know it was an issue

File: app/templates/public-services/index.hbs [hunk 2]
```diff
@@ -23,6 +23,17 @@
       @onChange={{this.handleSectorSelection}}
       @allowClear={{true}}
     />
+    <AuButton
+      @route="public-services.add"
```
[the remark this answers is not in the document]
Comment (Asergey91) [reply app/templates/public-services/index.hbs:27]: yep sry

**Comment by Asergey91** [comment]
> I assume the linting issues are caused by the target branch so we can ignore that for now.

yep, something about triple curly brackets

**Review by Windvis — APPROVED** [review]

File: app/templates/public-services/index.hbs [hunk 3]
```diff
@@ -23,6 +23,17 @@
...
     />
+    <AuButton
+      @icon="add"
+      @iconAlignment="left"
+      @skin="button"
+      @disabled={{this.addNewProduct.isRunning}}
+      @loading={{this.addNewProduct.isRunning}}
+      {{on "click" (perform this.addNewProduct)}}
+    >
+      Product toevoegen
```
Comment (cecemel) [inline app/templates/public-services/index.hbs:34]: Can you make 'Volledig nieuw product toevoegen'?

File: app/templates/public-services/index.hbs [hunk 4]
```diff
@@ -23,6 +23,17 @@
       @onChange={{this.handleSectorSelection}}
       @allowClear={{true}}
     />
+    <AuButton
```
Comment (cecemel) [inline app/templates/public-services/index.hbs:26]: The button is on the wrong page; should be where they can select templates

**Review by cecemel — CHANGES_REQUESTED** [review]

**Pull request merged**